backend/routes/patients.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+
```

Comment:
Please remove the blank lines. Keep at maximum one blank line for each section of code